### PR TITLE
174 backend implement full test 2

### PR DESCRIPTION
**Pull request by Ma-hawaj**

### CODE REVIEW

File: packages/functions/src/websockets/fullTest/autoSave.ts
Comment (mzhubail) [inline packages/functions/src/websockets/fullTest/autoSave.ts:1]: Well done! I'll try to make use of this.

**Comment by Ma-hawaj** [comment]
Currently the function that replaces the s3 keys with the presigned url implemented but not tested or used.
Waiting for the database branch to get merged to not face any errors.
Also the speaking grading needs to be tested .

**Comment by mzhubail** [comment]
I prefer we merge for now. and if there are any problems they will be solved in a new PR, where everything regarding full-test is in sync.

**Comment by Ma-hawaj** [comment]
Yes, that's find

**Pull request merged**